client/i18n/translations-schema.js

```diff
@@ -448,7 +448,7 @@ const translationsSchema = {
     email: 'Email',
     and: 'and',
     'change-theme': 'Sign in to change theme.',
-    'translation-pending': 'Help translate'
+    'translation-pending': 'Help us translate.'
```

Comment:
some translation-pending have . and some don't. do we need the or not?

Reply:
The reason is one is a sentence, as in a link in a sentence. And the other is a button label. We could re-visit this once its out.